ludwig/model_export/base_model_exporter.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+from ludwig.api import LudwigModel
+
+
+class BaseModelExporter(ABC):
+    def __init__(self, model):
+        self.model = model
+
+    @abstractmethod
+    def forward(self, x):
+        pass
+
+    @abstractmethod
+    def export_classifier(self, model_path, export_path, export_args_override):
+        pass
+
+    @abstractmethod
+    def quantize(self, path_fp32, path_int8):
```

Comment:
The variable naming assumes all non-quantized models to be fp32 and quantized to be int8. This is not always the case. For example, we can have fp16 => fp4 (https://youtu.be/MK4k64vY3xo?si=ULlFZQa_2DSvcgpX&t=1970)

Perhaps the function signature should be:
```
def quantize(self, model_path, quantized_model_path)
```